src/views/Login/Login.js

```diff
@@ -47,6 +47,19 @@ const SignIn = () => {
         event.preventDefault();
       };
 
+      const resetpassword = async () => {
+        toaster(t('Check your email to reset password'), 'success')
```

Comment:
This message should be commit from the backend after hitting the route as response because by this current implementation you tell a user to check email while request is not send yet. So what if request failed?